pom.xml

```diff
@@ -17,6 +17,29 @@
     <build>
         <plugins>
             <plugin>
+          		<groupId>org.apache.maven.plugins</groupId>
+          		<artifactId>maven-antrun-plugin</artifactId>
+          		<version>1.7</version>
+          		<executions>
+          		    <execution>
+          		        <phase>validate</phase>
+          		        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <echo>Build target: ${buildTarget}</echo>
+                                <echo>Service Locator provided by: ${serviceLocator.provider.groupName}.${serviceLocator.provider.artifactName}:${serviceLocator.provider.artifactVersion}</echo>
+                            </target>
```

Comment:
Is this necessary? I don't really see the point.

Reply:
This is just copied from the chirper project. We can remove it if it is not needed.